lib/modules/datasource/sbt-plugin/index.spec.ts

```diff
@@ -37,10 +38,12 @@ describe('modules/datasource/sbt-plugin/index', () => {
         .get('/maven2/org/scalatest/')
         .reply(
           200,
-          '<a href="scalatest/" title=\'scalatest/\'>scalatest_2.12/</a>\n' +
-            '<a href="scalatest_2.12/" title=\'scalatest_2.12/\'>scalatest_2.12/</a>\n' +
-            "<a href='scalatest_sjs2.12/'>scalatest_2.12/</a>" +
-            "<a href='scalatest_native2.12/'>scalatest_2.12/</a>",
+          codeBlock`
```

Comment:
```suggestion
          html`
```

use `html` template 😉 

https://www.npmjs.com/package/common-tags#html